### PR TITLE
fixing event title in grid changing back when editing twice

### LINKED ISSUE #50: Event title changes back to original one when editing twice or more

### PR DESCRIPTION
**Pull request by georgehrke**
fixes #50

### CODE REVIEW

**Comment by georgehrke** [comment]
please review @nextcloud/calendar

File: js/app/service/eventseditordialogservice.js [hunk 1]
```diff
@@ -24,145 +24,174 @@
...
+				context.openDialog(EDITOR_SIDEBAR, resolve, reject, unlock, [], scope, fcEvent, simpleEvent, calendar);
+			}
+		});
 
-			return self.promise;
+		return context.promise;
 	};
 
-	return this;
+	/**
```
Comment (tcitworld) [inline js/app/service/eventseditordialogservice.js]: What about these callbacks ?

Comment (georgehrke) [inline js/app/service/eventseditordialogservice.js]: So my IDE knows what parameters the callbacks [up there](https://github.com/nextcloud/calendar/pull/179/files#diff-a6601ae4e3cfb5427aee447d40ee87ccR140) expect

Comment (tcitworld) [inline js/app/service/eventseditordialogservice.js]: 👍

**Pull request merged**